input/pagecontent/changes.md

```diff
@@ -15,4 +15,5 @@ This change log documents the significant updates and resolutions implemented fr
   - added Must Support to ServiceRequest.priority [FHIR-46939](https://jira.hl7.org/browse/FHIR-46939)
   - AU eRequesting ServiceRequest profile renamed to be AU eRequesting Diagnostic Request [FHIR-46842](https://jira.hl7.org/browse/FHIR-46842)
 - Renamed AU eRequesting Patient Access actor and CapabilityStatement to be AU eRequesting Patient [FHIR-46802](https://jira.hl7.org/browse/FHIR-46802)
+- Changed requirements for population of ServiceRequest.code to specify an order of preference and removed AdditionalBinding in AU eRequesting Pathology Request and AU eRequesting Imaging Request [FHIR-46847](https://jira.hl7.org/browse/FHIR-46847)
```

Comment:
Don't think "to specify order of preference" is adding anything here. 
Suggest rewriting to something like: 
"Clarified profile-specific implementation guidance on populating ServiceRequest.code and removed additional binding in AU eRequesting Pathology Request and AU eRequesting Imaging Request [FHIR-46847](https://jira.hl7.org/browse/FHIR-46847)"
Or possibly split by the profile, like we do in AU Core.

Reply:
Amended on behalf of @johnscarter